addons/website/models/website.py

```diff
@@ -185,7 +185,7 @@ def _default_logo(self):
     custom_code_head = fields.Html('Custom <head> code', sanitize=False)
     custom_code_footer = fields.Html('Custom end of <body> code', sanitize=False)
 
-    robots_txt = fields.Html('Robots.txt', translate=False, groups='website.group_website_designer', sanitize=False)
+    robots_txt = fields.Html('Robots.txt', translate=False, groups='website.group_website_designer,website_blog.group_website_blog_manager', sanitize=False)
```

Comment:
Why you are access ```website_blog.group_website_blog_manager``` in website module? what is the need, if it is really needed then field should be overridden in website_blog module.

Reply:
Thank you for pointing this out! We will work on overriding the field in the `website_blog` module.